source/chapters/library.rst

```diff
@@ -471,6 +471,13 @@ Mixxx supports the following filters:
 
        ~bpm:100
 
+  * Constant BPM search finds tracks that were analyzed with "Assume constant tempo".
+    With "not"/negative prefix``-`` it finds all tracks with variable tempo.
+
+    ::
+
+       bpm:const|constant
+
 
```

Comment:
NIT:
```suggestion
  * Constant BPM search finds tracks that were analyzed with "Assume constant tempo".
    Combine with "not"/negative prefix``-`` to find all tracks with variable tempo.

    ::

       bpm:const
       bpm:constant
```

Reply:
oki, done.

Reply:
I also had reformulated the second sentence. Sorry, should've mentioned it since the diff doesn't highlight it.